examples/gitserver/README.md

```diff
@@ -38,6 +38,8 @@ Prerequisites:
     $ oc policy add-role-to-user edit -z git
     ```
 
+**NOTE:** To deploy the git server on OpenShift Online, you must switch to using persistent
+storage. See the *Persistent Storage* section below.
```

Comment:
On Tue, May 17, 2016 at 11:17 AM, Cesar Wong <notifications@github.com>
wrote:

> In examples/gitserver/README.md
> <https://github.com/openshift/origin/pull/8902#discussion_r63542087>:
>
> > @@ -38,6 +38,8 @@ Prerequisites:
> >      $ oc policy add-role-to-user edit -z git
> >      ```
> >
> > +**NOTE:** To deploy the git server on OpenShift Online, you must switch to using persistent
> > +storage. See the *Persistent Storage* section below.
>
> @bparees <https://github.com/bparees> had suggested that. But there's a
> couple of reasons that I don't like it but maybe you can convince me
> otherwise:
>
>    - The PVC claim size has to be adjusted to your environment's claim
>    size, which is 1G for online, but doesn't have to be that for other
>    environments.
>
> How is that different from our other templates?  By direction from Dan we
ended up going with the online default of 1G for those.

>
>    - The naming of things can get more complicated because now we'll have
>    dc/git-ephemeral vs dc/git-persistent, etc. If possible I'd like it to
>    remain short.
>
> I think our users are used to the convention.

>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8902/files/f7796cfc940a6305fef73e3c24b189c0856faff9#r63542087>
>